modules/theora/video_stream_theora.cpp

```diff
@@ -34,18 +34,8 @@
 #include "core/io/image.h"
 #include "core/os/os.h"
 #include "scene/resources/image_texture.h"
-
-#ifdef _MSC_VER
-#pragma warning(push)
-#pragma warning(disable : 4127)
-#endif
-
 #include "thirdparty/misc/yuv2rgb.h"
```

Comment:
Thirdparty includes should stay visually separate from core includes.
```suggestion

#include "thirdparty/misc/yuv2rgb.h"
```